state/migration_export.go

```diff
@@ -131,8 +131,10 @@ func (st *State) exportImpl(cfg ExportConfig) (description.Model, error) {
 	if err := export.modelUsers(); err != nil {
 		return nil, errors.Trace(err)
 	}
-	if err := export.machines(); err != nil {
-		return nil, errors.Trace(err)
+	if dbModel.Type() != ModelTypeCAAS {
```

Comment:
We really don't need this check. export.machines() is a no op for caas models. Having the check just adds complexity to the code.